Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! Fixing redirects through clck.yandex.ru
+||clck.yandex.ru^
```

Comment:
What about this? https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/df85668550d7ddc8e6bb71ab83e5102b8de4d816/Filters/exclusions.txt#L404

Reply:
Hm. It seems my PR is not required
![image](https://user-images.githubusercontent.com/8361299/126135843-8152d8a6-047d-4441-b357-0205dcb84156.png)